bciers/apps/reporting/tests/components/OperationReview.test.tsx

```diff
@@ -1,78 +1,65 @@
   });
 
-  it("renders the OperationReview component with given formData", async () => {
+  it("renders loading state initially", () => {
     render(
-      <OperationReview formData={mockFormData} version_id={mockVersionId} />,
+      <OperationReview
+        formData={null}
+        version_id={1}
+        reportingYear={{
+          reporting_year: 2024,
+          report_due_date: "2024-12-31",
+          reportingWindowEnd: "2024-12-31",
+        }}
+        allActivities={[]}
+        allRegulatedProducts={[]}
+      />,
     );
+    expect(screen.getByText("Loading...")).toBeInTheDocument();
+  });
 
-    // Check if the MultiStepHeader is rendered
-    expect(
-      screen.getAllByText(/Operation Information/i).length,
-    ).toBeGreaterThan(0);
-    expect(
-      screen.getAllByText(/Facilities Information/i).length,
-    ).toBeGreaterThan(0);
-    expect(screen.getAllByText(/Compliance Summary/i).length).toBeGreaterThan(
-      0,
-    );
-    expect(screen.getAllByText(/Sign-off & Submit/i).length).toBeGreaterThan(0);
-
-    // Check if the Review operation information header is rendered
-    expect(
-      screen.getAllByText(/Review operation information/i).length,
-    ).toBeGreaterThan(0);
-
-    // Check if the info message is rendered
-    expect(
-      screen.getByText(
-        /The information shown on this page is data entered in registration/i,
-      ),
-    ).toBeInTheDocument();
-
-    // Check if the form is rendered with correct data
-    expect(screen.getByLabelText(/Operator Legal Name/i)).toHaveValue(
-      "Test Operator Legal Name",
-    );
-    expect(screen.getByLabelText(/Operator Trade Name/i)).toHaveValue(
-      "Test Operator Trade Name",
-    );
-    expect(screen.getByLabelText(/Operation Name/i)).toHaveValue(
-      "Test Operation Name",
-    );
-    expect(screen.getByLabelText(/BCGHG ID/i)).toHaveValue("12345");
-    expect(screen.getByLabelText(/BORO ID/i)).toHaveValue("54321");
-    expect(screen.getByLabelText(/Operation Representative/i)).toHaveValue(
-      "Test Representative",
+  it("renders the form correctly after loading", async () => {
+    render(
+      <OperationReview
+        formData={{
+          activities: [1],
+          regulated_products: [1],
+          operation_representative_name: "John Doe",
+          operation_type: "Test Operation",
+        }}
+        version_id={1}
+        reportingYear={{
+          reporting_year: 2024,
+          report_due_date: "2024-12-31",
+          reportingWindowEnd: "2024-12-31",
+        }}
+        allActivities={[{ id: 1, name: "Activity 1" }]}
+        allRegulatedProducts={[{ id: 1, name: "Product 1" }]}
+      />,
     );
 
-    // Check if buttons are rendered
-    expect(
-      screen.getByText(/Sync latest data from registration/i),
-    ).toBeInTheDocument();
-    expect(screen.getByText(/Cancel/i)).toBeInTheDocument();
-    expect(screen.getByText(/Continue/i)).toBeInTheDocument();
```

Comment:
I think we can keep these tests! The functionality shouldn't have changed after the refactor